0004-ecs2ext-protocol/README.md

```diff
@@ -138,27 +123,21 @@ Example of status message from ECS to extension:
 }
 ```
 
-##### Message from extension to ECS
-Extension should send message to ECS using `reply to` NATS field value or to the next subject:
-```
-kaa.{protocol-version}.service.{service-instance-id}
-```
-Refer to [Kaa event-based protocol documentation](/) for `protocol-version` and `service-instance-id` parameters.
+#### Message from extension to ECS
+Extension should send message to ECS using `reply to` NATS field value or to the next subject: `kaa.v1.service.{service-instance-name}`.
 
-Format of the extension-to-ECS message:
-- `correlationId` (string, required) - refer to [Kaa event-based protocol documentation](/) for description.
-- `timestamp` (number, required) - message creation timestamp.
-- `timeout` (number, required) - amount of milliseconds from `timestamp`. Value '0' means "the request does not time out".
+In addition to `correlationId`, `timestamp`, and `timeout` fields that are defined in the [targeted messaging IPC design](/0003-messaging-ipc/README.md), extension-to-ECS message contains the following fields:
 - `appVersionToken` (string, required) - refer to [??](/) for description.
-- `extensionInstanceId` (string, required) - id of the instance that is message's source or destination. 
+- `extensionInstanceId` (string, required) - id of the instance that is message's source or destination.
```

Comment:
"`extensionInstanceId` (string, required) - id of"  must be replaced with "`extensionInstanceName` (string, required) - name of".